Signal/src/AppDelegate.m

```diff
@@ -176,7 +176,7 @@ - (UIViewController *)loadingRootViewController
     // we need to bump this constant.
     //
     // We added a number of database views in v2.13.0.
-    NSString *kLastVersionWithDatabaseViewChange = @"2.16.0";
+    NSString *kLastVersionWithDatabaseViewChange = @"2.13.0";
```

Comment:
We no longer need to rebuild this database view since we're lazy-deleting all of the existing dynamic interactions (except unread indicators which don't implement `OWSReadTracking` anyway.